### ISSUE TITLE
fidibus version display

### ISSUE THREAD

**Issue by vpbrendel** [BrendelGroup/AEGeAn#221]
seems to return v0+unknown - is this expected behavior?

**Comment by standage** [comment]
As a standalone module, GenHub/fidibus used the [versioneer](https://github.com/warner/python-versioneer) package for version handling. From its documentation:

> This is a tool for managing a recorded version number in distutils-based python projects. The goal is to remove the tedious and error-prone "update the embedded version string" step from your release process. Making a new release should be as easy as recording a new tag in your version-control system, and maybe making new tarballs.

So when I tagged a new version number by creating a new release on GitHub or running `git tag`, the `genhub.__version__` variable (which was printed when `fidibus --version` was run) was automatically updated.

I haven't looked at versioneer since fidibus was brought into AEGeAn as a submodule. I can't think of any reason it wouldn't work with the current setup, but it might need to be reinstalled and reconfigured. I'll take a look at this when I have a chance.

Disclaimer: I've got a proxy issue in my office at the moment that doesn't allow me to push to/pull from GitHub. I'll have to take a look at this at home.
